### ISSUE TITLE
Re-setup everytime run 2.3 and 2.4

### ISSUE THREAD

**Issue by cloudidn** [Genymobile/scrcpy#4890]
- [y] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [y] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS: Windows 10
 - scrcpy version: 2.3.1 and 2.4
 - installation method: Windows release
 - device model: HP Probook 445 G9
 - Android version: 11

**Describe the bug**
Every time I run the scrcpy it's always back to default resolution and bitrate settings, i have to re-setup the bitrate and resolution every time I launch it, I thought it require an update because last time I launch it was 2 months ago, so i install the 2.4 and I still have to re-setup it every time i launch it. Bear in mind that I don't know any code language, when I install 2.3.1 4 months ago it run perfectly without re-setup every launch, so I confused as hell about adb and whatnot, so please ELI5

**Comment by rom1v** [comment]
There is no "setup", scrcpy does not store any state between launches.

The parameters you pass are just for the current session. Btw, you can run several instance simultaneously on different devices (or even the same) with different parameters.

If you want to run always with the same parameters, create a script. On Windows, create `myscrcpy.bat` containing your full command, for example:

```
scrcpy --video-codec=h265 -b16M -m1920
```

https://github.com/Genymobile/scrcpy/blob/master/doc/windows.md#run

**Comment by adrianmmiller** [comment]
> There is no "setup", scrcpy does not store any state between launches.
> 
> The parameters you pass are just for the current session. Btw, you can run several instance simultaneously on different devices (or even the same) with different parameters.
> 
> If you want to run always with the same parameters, create a script. On Windows, create `myscrcpy.bat` containing your full command, for example:
> 
> ```
> scrcpy --video-codec=h265 -b16M -m1920
> ```
> 
> https://github.com/Genymobile/scrcpy/blob/master/doc/windows.md#run

be nice if that actually worked, none of the command line switches work as far as ive tried

**Comment by rom1v** [comment]
> be nice if that actually worked, none of the command line switches work as far as ive tried

How do you run them?

Try `scrcpy --m400` to make it obvious.
